extensions/amp-pan-zoom/0.1/amp-pan-zoom.js

```diff
@@ -303,12 +309,20 @@ export class AmpPanZoom extends AMP.BaseElement {
       height = this.sourceHeight_;
     }
 
+    const contentBox =
+      layoutRectFromDomRect(this.content_./*OK*/getBoundingClientRect());
+
     this.contentBox_ = layoutRectLtwh(
-        Math.round((this.elementBox_.width - width) / 2),
-        Math.round((this.elementBox_.height - height) / 2),
+        contentBox.left - this.elementBox_.left,
+        contentBox.top - this.elementBox_.top,
         Math.round(width),
         Math.round(height));
 
+    this.yOffsetFromCenter_ = this.contentBox_.top
+      - (this.elementBox_.height - this.contentBox_.height) / 2;
```

Comment:
lint didn't complain about this?! 😮 :)

Reply:
Lol. Let me see what I can do about making this more comprehensible for people who are similarly bad at math as myself.